app/views/shared/finalize.html.erb

```diff
@@ -0,0 +1,30 @@
+<div id="maincontent" class="row">
+
+  <% if @registration.expired? %>
+    <strong>Registration for <%= @registration.dsp_email %> has expired.
+            Please contact <%= mail_to ApplicationMailer.default[:from] %>.</strong>
+  <% elsif @already_used %>
+    <strong>This link is no longer valid; it may have already been used.</strong>
+  <% else %>
+    <p>Thank you.</p>
+    <p>Elevated access registration confirmed for <strong><%= @registration.dsp_email %></strong>.</p>
+    <p>A HathiTrust staff member will approve your registration and respond shortly by e-mail.</p>
+    
+    <% if @institution.entityID && @institution.shib_authncontext_class %>
+      <p>Your registration indicates you have access to mult-factor authentication (MFA).</p>
+      <p>You can test your ability to log in with multi-factor authentication using the link
+        <%= raw @institution.mfa_test_link %>.
+    <% elsif @registration.mfa_addendum? %>
+      <% # TODO HT-1212: we want to direct users out and back to pick up MFA info if possible %>
+      <p>Your registration indicates you have access to mult-factor authentication (MFA); however,
+         we do not currently have your institution configured for MFA access.
+         Please contact <%= mail_to ApplicationMailer.default[:from] %>.</p>
+    <% else %>
+      <p>Please take a moment to confirm that <code><%= raw @ip_address %></code> is a static IP address.
+         If you are not sure, we recommend you contact your network administrator.</p>
+      <p>You can test your ability to log in at HathiTrust using the link
+        <%= @institution.login_test_link %>.</p>
+      <pre><%= @whois.to_s %></pre>
```

Comment:
I had thought we were going to show the whois on the page where HT staff approved their registration -- was there a reason to show it here also? I would worry that it would be a little bit overwhelming for users since we can't easily parse it just to organization.

Reply:
Quoting the ticket, which I dutifully copy-pasted into the source file:  "show the IP address (and perhaps the WHOIS info?)" in reference to the submission page for the prospective user. But yeah it's pretty overwhelming. I haven't gotten as far as envisioning the UI for our staff to review the "signed agreements" which is where indeed some kind of whois info is called for (in a tab maybe?).